commcare-cloud/commcare_cloud/environment/schemas/fab_settings.py

```diff
@@ -0,0 +1,24 @@
+import jsonobject
+
+GitUriProperty = jsonobject.StringProperty
+TimezoneProperty = jsonobject.StringProperty
```

Comment:
what's the purpose of overriding these?

Reply:
Signals to the reader what values types are supposed to be here. Makes relatively explicit, for example, the validation code someone would write if they wanted to tighten up the schema even more. I've done this in other schemas and usually been happy I have when I'm revisiting the code to remember what various properties are/look like.